loom/common/xml/tinyxml2.h

```diff
@@ -1150,10 +1160,10 @@ class XMLDocument : public XMLNode
 	const char* errorStr2;
 	char* charBuffer;
 
-	MemPoolT< sizeof(XMLElement) >	elementPool;
-	MemPoolT< sizeof(XMLAttribute) > attributePool;
-	MemPoolT< sizeof(XMLText) >		textPool;
-	MemPoolT< sizeof(XMLComment) >	commentPool;
+    MemPoolT< TIXML2_MEM_ALIGN(sizeof(XMLElement)) >	elementPool;

```

Comment:
Oh - yeah if the changes to tinyxml are ultimately do nothing we should revert them rather than keep unused code that will bitrot.